app/models/quiz.rb

```diff
@@ -4,4 +4,6 @@ class Quiz < ApplicationRecord
   has_many :answer_options, through: :quiz_questions
 
   validates :title, presence: true
+
+  validates_with RateLimitValidator, limit: 5, scope: :target_id
```

Comment:
A target can have only one quiz. Validate uniqueness instead.

Reply:
I have removed the rate limiting and created an issue tackle the uniqueness validation. https://github.com/pupilfirst/pupilfirst/issues/1431